### PR TITLE
mons: add command to restore mon quorum

### PR DESCRIPTION
**Pull request by subhamkrai**
This commits add `mons restore-quorum` command
to restore mon quorum based on single good mon.
Also adding docs for the commnad.

### CODE REVIEW

**Comment by subhamkrai** [comment]
Local testing works as expected.

File: cmd/odf/mon/mon.go [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package mon
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// MonsCmd represents the mon command
+var MonsCmd = &cobra.Command{
+	Use:                "mons",
+	Short:              "Supports sub-commands like 'restore-quorum'",
```
Comment (travisn) [inline cmd/odf/mon/mon.go:10]: I'd suggest not to mention specific subcommands, just keep it general
```suggestion
	Short:              "Supports sub-commands for mons'",
```

File: cmd/odf/mon/mon.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+	DisableFlagParsing: true,
+	Hidden:             true,
```
Comment (travisn) [inline cmd/odf/mon/mon.go:12]: All of the mon commands are hidden? I was thinking only advanced commands like restore-quorum might need to be hidden.

File: cmd/odf/mon/restore_quorum.go [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+	"github.com/spf13/cobra"
+)
+
+// RestoreQuorum represents the mons command
+var RestoreQuorum = &cobra.Command{
+	Use:                "restore-quorum",
+	Short:              "When quorum is lost, restore quorum to the remaining healthy mon",
+	DisableFlagParsing: true,
+	Args:               cobra.ExactArgs(1),
+	Example:            "odf mons restore-quorum <Mon_ID>",
```
Comment (travisn) [inline cmd/odf/mon/restore_quorum.go:15]: nit: CLI examples are better lower-case IMO
```suggestion
	Example:            "odf mons restore-quorum <mon_id>",
```

**Review by travisn — CHANGES_REQUESTED** [review]
a few small suggestions

File: cmd/odf/mon/mon.go (hunk 1 above)
Comment (subhamkrai) [reply cmd/odf/mon/mon.go]: got it

File: cmd/odf/mon/mon.go (hunk 2 above)
Comment (subhamkrai) [reply cmd/odf/mon/mon.go:12]: IIRC, Eran's comment was to have all the dev preview command hidden. I guess it is also mentioned in Jira by him

Comment (travisn) [reply cmd/odf/mon/mon.go:12]: Let's discuss this more. I see the point of hiding dev preview items, but also would prefer not to.

**Review by travisn — APPROVED** [review]

**Pull request merged**